modules/ROOT/pages/on-error-scope-concept.adoc

```diff
@@ -65,11 +65,56 @@ to fatal errors, which have error messages that contain the word "fatal":
 ----
 
 Note that matching conditions are evaluated sequentially, in the order in
-which the On-Error components reside in the Error Handler. So, for example, if
-you want to handle specific HTTP errors in one way but handle all other errors
-a different way, you need to provide an On-Error component configuration for
-the HTTP errors before a component that captures the remaining errors with
-ANY. Keep in mind that default error handling takes place for errors you
+which the On-Error components reside in the Error Handler. For example, if
+you want to handle `VALIDATION:NOT_NULL` in one way but handle all other
+errors a different way, you need to provide an On-Error component configuration
+for that error before a component that captures the remaining errors
+(identified as `ANY` by the second On-Error component). The following error
+handlers are configured to produce the
+`{ "MyError": "value was expected to be null" }` for a `VALIDATION:NOT_NULL`
+error, rather than the `{ "messageANY" : "Some other error" }`. The string
+`value was expected to be null` is the description of the
+`VALIDATION:NOT_NULL` error. Such an error occurs if your Mule application
```

Comment:
This paragraph is a bit hard to understand. Maybe we can just say that the error is thrown by the "Is Null" validation component and skip that the application must contain it?

Reply:
revised

Reply:
revised